### PR TITLE
[ZT] update google workspace instructions

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/identity/idp-integration/gsuite.md [hunk 1]
```diff
@@ -6,123 +6,80 @@ weight: 13
...
+    4. The summary page will load and you can save and exit.
 
-    The Client ID will now appear in the `APIs & Services` page.
+9. Return to the **Credentials** page and select **+ Create Credentials** > **OAuth client ID**.
 
-    ![Client ID Visible](/cloudflare-one/static/documentation/identity/gsuite/client-id-visible.png)
+    ![Location of OAuth client ID settings on Google Cloud Platform credentials page.](/cloudflare-one/static/documentation/identity/google/create-oauth.png)
 
-17. On the Zero Trust dashboard, navigate to **Settings > Authentication**.
+10. Choose _Web application_ as the Application type.
```
Comment (marciocloudflare) [inline content/cloudflare-one/identity/idp-integration/gsuite.md:47]: Is this a drop-down menu? Italics are reserved for those :)

File: content/cloudflare-one/tutorials/google-workspace-saas.md [hunk 2]
```diff
@@ -1,60 +1,67 @@
...
-1.  Follow [this guide](https://support.google.com/a/answer/6349809?hl=en&ref_topic=7556907) to collect the Entity ID and Assertion Consumer Service URL to add to your application:
+5. On the next page, you will see your **SSO endpoint**, **Access Entity ID or Issuer**, and **Public Key**. These values will be used to configure Google Workspace.
 
-    - Entity ID: `google.com`
-    - Assertion Consumer Service URL: `https://google.com/a/your_domain.com/acs`
-    - Name ID: `Email`
+## 2. Create a certificate from your Public Key
 
-    ![Add fields to the Zero Trust application](/cloudflare-one/static/zero-trust-security/google-workspace-saas/teams-app-fields.png)
+1. Copy paste your **Public Key** into a text editor.
```
Comment (marciocloudflare) [inline content/cloudflare-one/tutorials/google-workspace-saas.md]:
```suggestion
1. Copy and then paste your **Public key** into a text editor.
```

File: content/cloudflare-one/tutorials/google-workspace-saas.md [hunk 3]
```diff
@@ -1,60 +1,67 @@
...
+2. Wrap the certificate in `-----BEGIN CERTIFICATE-----` and `-----END CERTIFICATE-----`. For example,
 
-1.  Click **Next** to create an Access policy and use the following mapping to set up your Workspace sign-in:
+    ```txt
+    -----BEGIN CERTIFICATE-----
+    <Public key>
```
Comment (marciocloudflare) [inline content/cloudflare-one/tutorials/google-workspace-saas.md]:
```suggestion
    <PUBLIC_KEY>
```

File: content/cloudflare-one/tutorials/google-workspace-saas.md [hunk 4]
```diff
@@ -1,60 +1,67 @@
...
-1.  Next, you can select **Use a domain specific issuer**. If you select this option, Google will send an issuer specific to your domain (where `your_domain.com` is replaced with your actual primary Google Workspace domain name).
+## 3. Create an SSO provider in Google Workspace
 
-## Create and upload a certificate file
+1. Log in to your [Google Admin console](https://admin.google.com/).
+2. Navigate to **Security** > **Authentication** > **SSO with third party IdP**.
+3. Select **Third-party SSO profile for your organization**.
+4. Enable **Set up SSO with third-party identity provider**.
+5. Fill in the following information:
+    - **Sign-in page URL**: Copy paste your **SSO Endpoint** from the Zero Trust dashboard.
```
Comment (marciocloudflare) [inline content/cloudflare-one/tutorials/google-workspace-saas.md]:
```suggestion
    - **Sign-in page URL**: Copy and then paste your **SSO endpoint** from the Zero Trust dashboard.
```

File: content/cloudflare-one/tutorials/google-workspace-saas.md [hunk 5]
```diff
@@ -1,60 +1,67 @@
...
+    - **Sign-out page URL**: `https://<team-name>.cloudflareaccess.com/cdn-cgi/access/logout`, where `<team-name>` is your Zero Trust [team name](/cloudflare-one/glossary/#team-name).
+    - **Verification certificate**: Upload the certificate file containing your Public Key.
```
Comment (marciocloudflare) [inline content/cloudflare-one/tutorials/google-workspace-saas.md]:
```suggestion
    - **Verification certificate**: Upload the certificate file containing your public key.
```

Comment (marciocloudflare) [reply content/cloudflare-one/tutorials/google-workspace-saas.md]: Changing it here, as public key is the regular noun, and not the option customers see on a dash. Wdyt?

File: content/cloudflare-one/tutorials/google-workspace-saas.md [hunk 6]
```diff
@@ -1,60 +1,67 @@
...
-
-You can now test the integration by going to `https://google.com/a/<yourdomain>`.
+To test the integration, open an incognito browser window and go to `https://mail.google.com/`. An Access login screen should appear.
 
 ## Troubleshooting
 
 `Error: “G Suite - This account cannot be accessed because the login credentials could not be verified.”`
 
-If you see this error, it is likely that the Public Key and Private Key do not match. Confirm your certificate file matches your Public Key.
+If you see this error, it is likely that the Public Key and Private Key do not match. Confirm that your certificate file includes the correct Public Key.
```
Comment (marciocloudflare) [inline content/cloudflare-one/tutorials/google-workspace-saas.md]:
```suggestion
If you see this error, it is likely that the public key and private key do not match. Confirm that your certificate file includes the correct public key.
```

Comment (marciocloudflare) [reply content/cloudflare-one/tutorials/google-workspace-saas.md]: Unless these are options on the dashboard that are title cased, I think we can use regular lowercase.

**Review by marciocloudflare — APPROVED** [review]

File: content/cloudflare-one/tutorials/google-workspace-saas.md [hunk 7]
```diff
@@ -1,60 +1,67 @@
...
-## Create an application on the Zero Trust Dashboard
+{{<Aside type="warning">}}
+When you put your Google Workspace behind Access, users will not be able to log in using [Google](/cloudflare-one/identity/idp-integration/google/) or [Google Workspace](/cloudflare-one/identity/idp-integration/gsuite/) as an identity provider.
+{{</Aside>}}
 
-1.  On the Zero Trust Dashboard, navigate to **Access** > **Applications** and create a SaaS application.
+4. On the next page, [create an Access policy](/cloudflare-one/policies/access/) for your application. For example, you could allow users with an `@your_domain.com` email address.
 
-1.  Follow [this guide](https://support.google.com/a/answer/6349809?hl=en&ref_topic=7556907) to collect the Entity ID and Assertion Consumer Service URL to add to your application:
+5. On the next page, you will see your **SSO endpoint**, **Access Entity ID or Issuer**, and **Public Key**. These values will be used to configure Google Workspace.
```
Comment (ranbel) [inline content/cloudflare-one/tutorials/google-workspace-saas.md]:
```suggestion
5. On the next page, you will see your **SSO endpoint**, **Access Entity ID or Issuer**, and **Public key**. These values will be used to configure Google Workspace.
```

File: content/cloudflare-one/tutorials/google-workspace-saas.md [hunk 8]
```diff
@@ -1,60 +1,67 @@
...
 
-    - Entity ID: `google.com`
-    - Assertion Consumer Service URL: `https://google.com/a/your_domain.com/acs`
-    - Name ID: `Email`
+## 2. Create a certificate from your Public Key
```
Comment (ranbel) [inline content/cloudflare-one/tutorials/google-workspace-saas.md]:
```suggestion
## 2. Create a certificate from your public key
```

File: content/cloudflare-one/identity/idp-integration/gsuite.md (hunk 1 above)
Comment (ranbel) [reply content/cloudflare-one/identity/idp-integration/gsuite.md:47]: yes, this is a drop-down menu in the Google Cloud dashboard.

**Pull request merged**